Makefile

```diff
@@ -88,6 +73,10 @@ $(LINT_BIN):
 	@$(call print, "Fetching gometalinter.v2")
 	GO111MODULE=off go get -u $(LINT_PKG)
 
+$(GOACC_BIN):
+	@$(call print, "Fetching go-acc")
+	$(GOINSTALL) $(GOACC_PKG)
```

Comment:
Is this what users expect? That they get a tool installed automatically?

Reply:
Or should we now start with `make toolchain` to install this and the grpc stuff as well. And then remove the autogen. code from the repo.

Reply:
this is really only for travis builds, so i think it's okay in this case

Reply:
ok